### ISSUE TITLE
stock 'freebsd/FreeBSD-11.0-CURRENT' box provision failure, sudo unable to load libpam.so.6

### ISSUE THREAD

**Issue by jhgorse** [brd/packer-freebsd#16]
vagrant up --no-destroy-on-error
```
==> default: Setting hostname...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

# Set the hostname
hostname 'openafs-freebsd-dev'
sed -i '' 's/^hostname=.*$/hostname="openafs-freebsd-dev"/' /etc/rc.conf

# Remove comments and blank lines from /etc/hosts
sed -i'' -e 's/#.*$//' /etc/hosts
sed -i'' -e '/^$/d' /etc/hosts

# Prepend ourselves to /etc/hosts
grep -w 'openafs-freebsd-dev' /etc/hosts || {
  echo -e '127.0.0.1\topenafs-freebsd-dev\topenafs-freebsd-dev' | cat - /etc/hosts > /tmp/tmp-hosts
  mv /tmp/tmp-hosts /etc/hosts
}


Stdout from the command:



Stderr from the command:

sudo: error in /usr/local/etc/sudo.conf, line 0 while loading plugin `sudoers_policy'
sudo: unable to load /usr/local/libexec/sudo/sudoers.so: Shared object "libpam.so.6" not found, required by "sudoers.so"
sudo: fatal error, unable to load plugins
```

vagrant ssh
```
vagrant@:~ % sudo
sudo: error in /usr/local/etc/sudo.conf, line 0 while loading plugin `sudoers_policy'
sudo: unable to load /usr/local/libexec/sudo/sudoers.so: Shared object "libpam.so.6" not found, required by "sudoers.so"
sudo: fatal error, unable to load plugins
vagrant@:~ % su root
...
root@:/home/vagrant # sudo
sudo: error in /usr/local/etc/sudo.conf, line 0 while loading plugin `sudoers_policy'
sudo: unable to load /usr/local/libexec/sudo/sudoers.so: Shared object "libpam.so.6" not found, required by "sudoers.so"
sudo: fatal error, unable to load plugins
```

**Comment by jhgorse** [comment]
Also, for reference, my Vagrantfile
```rb
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|
  config.vm.guest = :freebsd
  config.vm.box = "freebsd/FreeBSD-11.0-CURRENT"
  config.vm.hostname = "openafs-freebsd-dev"
  config.vm.synced_folder ".", "/vagrant", id: "vagrant-root", disabled: true
  config.vm.base_mac = "080027D14C66"
  config.ssh.shell = "sh"
  ["vmware_fusion", "vmware_workstation", "virtualbox"].each do |provider|
    config.vm.provider provider do |v, override|
      host = RbConfig::CONFIG['host_os']
      v.memory = get_memory_setting(host)
      v.cpus = get_cpu_setting(host)
    end
  end
  config.vm.provider :virtualbox do |vb|
    vb.customize ["modifyvm", :id, "--hwvirtex", "on"]
    vb.customize ["modifyvm", :id, "--audio", "none"]
    vb.customize ["modifyvm", :id, "--nictype1", "virtio"]
    vb.customize ["modifyvm", :id, "--nictype2", "virtio"]
  end

  # config.vm.provision "shell", path: "provision_script.sh"
end

# Use 1/divider of the available memory
# TODO: use 1/divider or 2 GB, whichever is larger
def get_memory_setting(host)
  divider = 4
  if host =~ /darwin/
    mem = `sysctl -n hw.memsize`.to_i / 1024 / 1024 / divider
  elsif host =~ /linux/
    mem = `grep 'MemTotal' /proc/meminfo | sed -e 's/MemTotal://' -e 's/ kB//'`.to_i / 1024 / divider
  else # Windows
    mem = `for /F "tokens=2 delims==" %i in ('wmic computersystem get TotalPhysicalMemory /value') do @echo %i`.to_i / 1024 / 1024 / divider
  end
  return mem
end

def get_cpu_setting(host)
  if host =~ /darwin/
    cpus = `sysctl -n hw.ncpu`.to_i
  elsif host =~ /linux/
    cpus = `nproc`.to_i
  else # Windows
    cpus = `for /F "tokens=2 delims==" %i in ('wmic cpu get NumberOfCores /value') do @echo %i`.to_i
  end
  return cpus
end
```

**Comment by jhgorse** [comment]
Fixed in freebsd/FreeBSD-11.0-BETA3
https://atlas.hashicorp.com/FreeBSD/
